src/Core/Managed/Shared/Extensibility/Implementation/Property.cs

```diff
@@ -17,15 +19,42 @@ internal static class Property
         public const int MaxDictionaryNameLength = 150;
         public const int MaxDependencyTypeLength = 1024;
         public const int MaxValueLength = 8 * 1024;
+        public const int MaxResultCodeLength = 1024;
         public const int MaxEventNameLength = 512;
         public const int MaxNameLength = 1024;
         public const int MaxMessageLength = 32768;
         public const int MaxUrlLength = 2048;
-        public const int MaxCommandNameLength = 2 * 1024;
+        public const int MaxDataLength = 8 * 1024;
         public const int MaxTestNameLength = 1024;
         public const int MaxRunLocationLength = 2024;
         public const int MaxAvailabilityMessageLength = 8192;
 
+        public static readonly IDictionary<string, int> TagSizeLimits = new Dictionary<string, int>()
```

Comment:
Why do these need to be in a dictionary as opposed to constants?
Do we need to look them up at runtime by name, as done in SanitizeTelemetryContext currently, or do we have enough control to generate out the specific property references so we don't pay the hashing and lookup costs?